blockscout-ens/bens-proto/proto/bens.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package blockscout.bens.v1;
+
+option go_package = "github.com/blockscout/blockscout-rs/bens";
+
+service DomainsExtractor {
+  rpc SearchAddress(SearchAddressRequest) returns (SearchAddressResponse) {}
+  rpc GetDetailedDomain(GetDetailedDomainRequest) returns (GetDetailedDomainResponse) {}
+}
+
+enum DomainSource {
+  /// https://docs.ens.domains/
+  ENS = 0;
+  // /// https://docs.unstoppabledomains.com/
+  //UNSTOPPABLE_DOMAINS = 1;
+}
+
+message Domain {
+  /// Source of domain name. Now could be only ENS, in future we could add UNSTOPPABLE_DOMAINS
+  DomainSource domain_source = 1;
```

Comment:
Let maybe add this field when another domain will be added? It seems to be non-breaking change after all